client/src/components/game.js

```diff
@@ -2,33 +2,32 @@ import { useState, useEffect } from "react";
 import QuestionCard from "./questioncard";
 
 const Game = (props) => {
+  const [questions, setQuestions] = useState([]);
 
-    const [questions, setQuestions] = useState([]);
+  //make sure to have the backend server be consistent
+  const loadData = () => {
+    fetch("http://localhost:5000/quizgame")
+      .then((response) => response.json())
+      .then((data) => {
+        console.log("This is line 11", data.data);
+        setQuestions(data.data);
+      });
+  };
 
-    const loadData = () => {
-        fetch('http://localhost:5000/api/game')
-            .then((response) => response.json())
-            .then(data => {
-                console.log("This is line 11", data.results);
-                setQuestions(data.results);
-            })
-    }
+  useEffect(() => {
+    loadData();
```

Comment:
🔴 If you're going to fetch data with the `useEffect` hook, you should follow this pattern to avoid bugs: https://beta.reactjs.org/learn/synchronizing-with-effects#fetching-data